docs(sources): add integrations directory with 31 source pages (ETL-980) (#739)
glassflow/clickhouse-etl: docs/app/sources/cloudwatch/page.mdx

---
title: 'AWS CloudWatch'
description: 'Ingest logs and metrics from AWS CloudWatch into ClickHouse with GlassFlow'
---
import { Callout } from 'nextra/components'

# AWS CloudWatch

CloudWatch is AWS's monitoring and observability service that collects logs, metrics, and events from cloud resources. It provides alerting, dashboards, and insights into system performance and health.

## Using AWS CloudWatch with GlassFlow

GlassFlow can ingest logs and metrics from AWS CloudWatch into ClickHouse with the same pipeline capabilities available to all GlassFlow sources — deduplication, filtering, and stateless transformations.

<Callout type="info">

> **Available on request.** AWS CloudWatch connectors are supported natively via the GlassFlow Enterprise Edition. To get started, [join our GlassFlow Slack community](https://glassflowhub.slack.com/join/shared_invite/zt-349m7lenp-IFeKSGfQwpJfIiQ7oyFFKg) to chat directly with our team, or [reach out](https://www.glassflow.dev/integrations#contact) to us with your use case, expected volumes, and timeline.

Reviewer: ```suggestion
**Available on request.** AWS CloudWatch connectors are natively supported in GlassFlow Enterprise Edition. To get started, [join our GlassFlow Slack community](https://glassflowhub.slack.com/join/shared_invite/zt-349m7lenp-IFeKSGfQwpJfIiQ7oyFFKg) to chat directly with our team, or [reach out](https://www.glassflow.dev/integrations#contact) to us with your use case, expected volumes, and timeline.
```
